.github/workflows/api-dotnetcore.yml

```diff
@@ -7,7 +7,24 @@ on:
     branches: [master, dev, test]
 
 jobs:
+  # JOB to run change detection
```

Comment:
This is nice, thanks for this.

Reply:
Could we do something like this for Openshift as well do you think? Ie. skipping builds if not required or only checking out the parts of the repo that we need.

Reply:
I believe so. We can chat some more about to improve it